_episodes/12-cluster.md

```diff
@@ -46,6 +47,14 @@ tasks.
 
 
 ## Logging in
+The first step in using a cluster is to establish a connection from our laptop to the cluster,
+via the Internet and/or your organisation's network. We use a program called the 
+Secure SHell (or ssh) client for this. (make sure you have a SSH client installed on your
```

Comment:
'`this. (make`' -> should be '`this (make ... details).`' or '`this. Make ... details.`'

Reply:
278c329